extensions/wikia/WikiaHomePage/js/WikiaHomePage.js

```diff
@@ -535,8 +543,8 @@ WikiaHomePageRemix.prototype = {
 		if( this.heroImage !== null ) {
 			img = $(this.heroImage)
 				.attr('alt', imgData.title)
-				.attr('witdh', imgData.width)
-				.attr('witdh', imgData.height);
+				.attr('witdh', this.SPONSOR_HERO_IMAGE_WIDTH)
```

Comment:
Where? It actually WAS typo here. If you mean double `width` here: `.attr('witdh', imgData.width).attr('witdh', imgData.height);`